nilearn/decomposition/tests/test_canica.py

```diff
@@ -3,17 +3,34 @@
 import numpy as np
 from numpy.testing import assert_array_almost_equal
 from nose.tools import assert_true, assert_raises
-
 import nibabel
-
 from nilearn.decomposition.canica import CanICA
 
 
-def test_canica_square_img():
-    shape = (20, 20, 1)
-    affine = np.eye(4)
-    rng = np.random.RandomState(0)
+try:
+    from nose.tools import assert_less_equal
+except ImportError:
+    def assert_less_equal(a, b):
+        if a > b:
+            raise AssertionError("%f is not less than %f" % (a, b))
+
+
+def _make_data_from_components(components, affine, shape, rng=None,
```

Comment:
Function name, even if private, could be more explicit, particularly with respect to the format of the output.